### PR TITLE
Drag and Drop with Auth

### PR DESCRIPTION
**Pull request by gizzmojr**
Here's first go at drag and drop. Needs visual work and some UI enhancements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,6 +42,26 @@
...
+                border-width: 1;
+	        }	
+            #RESOLVED {
+                border-style: solid;
+                border-width: 1;
+	        }	
+            #VERIFIED {
+                border-style: solid;
+                border-width: 1;
+	        }
```
Comment (leif81) [inline index.html:75]: Ah, can you rebase onto master, I did some card styling last night that actually does the same thing so all these won't be needed anymore.

File: index.html [hunk 2]
```diff
@@ -254,12 +343,188 @@
...
+                        var response = xhr.responseText;
+                        if (response == "") { // Working response is "5.0" for example
+                            alert("Unable to access Bugzilla REST : " + bzURL);
+                            return;
+                        }
+                    }
+                }
+                xhr.open("GET", bzURL + "/rest/version");
+                xhr.send();
+	    }
```
Comment (leif81) [inline index.html]: checkURL should go away too once you rebase on master.

**Comment by leif81** [comment]
I'm going to try rebasing on my side. Seems like some code that should have been gone reappeared in some of the commits on the auth branch.

**Comment by gizzmojr** [comment]
That is more then possible, I've rebased it so many times with quite a few changes that doing late night coding left some conflicts in I'm sure of it.

**Comment by leif81** [comment]
:)

File: index.html (hunk 1 above)
Comment (gizzmojr) [inline index.html:75]: These aren't for the cards, they're for the div's of the board columns. That way when someone drags they know where each column is. Right now when scrolled down, and the headers are out of view, hard to tell where the correct target column starts/begins.

Comment (leif81) [inline index.html:75]: Ah Ok I get it now.

**Comment by gizzmojr** [comment]
New rebase off master, and tweaked the drop item code so it's no longer looking at card elements.

**Comment by gizzmojr** [comment]
Need to not hard-code the resolution, as mentioned offline you have a great UI design to simplify that. 

Possibly add a Log Off button (currently tab session based storage with auth)

**Comment by leif81** [comment]
Thanks Matt. Lots of nice work here. I'll tidy up a few things post merge.

**Pull request merged**